.github/workflows/playwright.yml

```diff
@@ -210,21 +171,40 @@ jobs:
     - uses: actions/upload-artifact@v4
       if: always()
       with:
-        name: test-results-ubuntu-${{ github.sha }}
-        path: test-results/
+        name: test-results-ubuntu-${{ matrix.shardIndex }}-${{ github.sha }}
+        path: |
+          test-results/
+          playwright-report/
         retention-days: 30
         overwrite: true
+    - name: Run ubuntu/chrome flow (retry 1)
+      run: yarn playwright test --project="Google Chrome" --last-failed --pass-with-no-tests
```

Comment:
`pass-with-no-tests` this is required so that the step does not fail in situation where the initial attempt passes and does not require a retry